.github/workflows/smbios_ci.yml

```diff
@@ -13,7 +13,10 @@ env:
 jobs:
   build:
     if: ${{ github.event.label.name == 'Ready for Review' }} || github.event_name == 'push'
-    runs-on: windows-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
> Will this break the Readme.md badges?

Yes, I'll. I just found out that. It will report as a whole, but not individual builds. Seems like this is an issue for the matrix os case. Saw other comments asking for this enhancement.

Reply:
Are you able to edit readme.md to report as a whole?  I am guessing it will break the readme.md badges if the badges are not removed or replaced with a single one (if possible)?

Reply:
> Are you able to edit readme.md to report as a whole? I am guessing it will break the readme.md badges if the badges are not removed or replaced with a single one (if possible)?

Right, I need to fix the `readme.md` and have only one badge